packages/client/lib/OpenID4VCIClient.ts

```diff
@@ -136,7 +137,46 @@ export class OpenID4VCIClient {
     });
   }
 
-  private handleAuthorizationDetails(authorizationDetails?: AuthDetails | AuthDetails[]): AuthDetails | AuthDetails[] | undefined {
+  public async acquirePushedAuthorizationRequestURI({
+    clientId,
+    codeChallengeMethod,
+    codeChallenge,
+    authorizationDetails,
+    redirectUri,
+    scope,
+  }: AuthRequestOpts): Promise<OpenIDResponse<unknown>> {
```

Comment:
Why the unknown? You know what payload type to expect right?

Reply:
Fixed!